cg/apps/lims/orderform.py

```diff
@@ -120,14 +130,59 @@ def get_project_type(document_title: str, parsed_samples: List) -> str:
         if len(analyses) != 1:
             raise OrderFormError(f"mixed 'Data Analysis' types: {', '.join(analyses)}")
 
-        project_type = analyses.pop()
-        if "mip" in project_type and "balsamic" in project_type:
-            raise OrderFormError(f"mixed 'Data Analysis' types: {project_type}")
+        analysis = analyses.pop()
+
+        if analysis == "no analysis":
+            project_type = "fastq"
+        else:
+            project_type = analysis
 
     return project_type
 
 
-def expand_case(case_id: str, parsed_case: dict) -> tuple:
+def get_data_delivery(samples: [dict], project_type: OrderType) -> str:
+    """Determine the order_data delivery type."""
+
+    data_deliveries = parse_data_deliveries(samples)
```

Comment:
Please add type hint, so much easier to read the code

Reply:
fixed